site/_posts/2017-02-27-embedding-ansible-additions.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Last Week in ManageIQ: Embedding Ansible Additions"
+author: jjlangholtz
+date: 2017-02-28
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Greetings everyone! [Josh](https://github.com/jjlangholtz) here with yet another weekly update! :tada:
+
+## Featured
+
+Another outstanding week with over 109 PRs merged. Here is the breakdown:
+
+* [49 from ManageIQ core][manageiq PRs merged]
+* [42 from the Classic UI][manageiq-ui-classic PRs merged]
+* [13 from the Service UI][manageiq-ui-service PRs merged]
+* [5 from the Amazon Provider][manageiq-providers-amazon PRs merged]
+
+## Improved
+
+There were quite a few improvements to facilitate the embedding Ansible effort. First, [Ladas](https://github.com/Ladas) has laid the foundation for [query optimizations](https://github.com/ManageIQ/manageiq/pull/13978) for inventory collections based on tracking graph dependencies. [Nick](https://github.com/carbonin) implemented a [worker lifecycle for embedded Ansible](https://github.com/ManageIQ/manageiq/pull/14023) to aid monitoring efforts. To simplify decorating models, [Martin](https://github.com/himdel) has added an implementation for [class decorator support](https://github.com/ManageIQ/manageiq-ui-classic/pull/237) which also removes the need to use the draper gem.
```

Comment:
I don't believe these are all related to the Ansible effort described in the first sentence, are they?  The first sentence makes it seem like the sentences following it are for PRs in the Ansible area.  I could be wrong but I don't think the decorators are.. maybe not even the query optimizations?